### PR TITLE
TASK-357: Add unsaved notifications to all update and create forms

### PR DESCRIPTION
**Pull request by Rodielm**
Display a warning message about unsaved data when the user creates or updates information.

### CODE REVIEW

**Comment by Rodielm** [comment]
Task linked: [TASK-357 Add unsaved notifications to all update and create forms](https://app.clickup.com/t/36631768/TASK-357)

File: src/webcomponents/clinical/clinical-analysis-create.js [hunk 1]
```diff
@@ -340,6 +340,15 @@ export default class ClinicalAnalysisCreate extends LitElement {
                 {
                     title: "General Information",
                     elements: [
+                        {
+                            title: "",
```
Comment (jmjuanes) [inline src/webcomponents/clinical/clinical-analysis-create.js:344]: Please remove the `title` field if is not used.

File: src/webcomponents/sample/sample-update.js [hunk 2]
```diff
@@ -301,11 +312,10 @@ export default class SampleUpdate extends LitElement {
                         display: {
                             render: source => html`
                                 <external-source-update
-                                    .source="${source}"
+                                    .source=${source}
```
Comment (jmjuanes) [inline src/webcomponents/sample/sample-update.js:315]: Please undo this change, attributes should be always wrapped in double quotes.

File: src/webcomponents/study/ontology-term-annotation/ontology-term-annotation-update.js [hunk 3]
```diff
@@ -109,12 +121,12 @@ export default class OntologyTermAnnotationUpdate extends LitElement {
             <data-form
                 .data=${this.ontology}
                 .config="${this._config}"
-                .updateParams=${this.updateParams}
+                .updateParams=${this.data?.updateParams}
```
Comment (jmjuanes) [inline src/webcomponents/study/ontology-term-annotation/ontology-term-annotation-update.js:124]: Please wrap this attribute value in double quotes.

File: src/webcomponents/study/status/status-update.js [hunk 4]
```diff
@@ -54,10 +54,17 @@ export default class StatusUpdate extends LitElement {
         this._config = this.getDefaultConfig();
     }
 
-    update(changedProperties) {
+    firstUpdated(changedProperties) {
         if (changedProperties.has("status")) {
+            console.log("update status ...");
```
Comment (jmjuanes) [inline src/webcomponents/study/status/status-update.js:59]: Please remove this `console.log` line.

**Review by jmjuanes — CHANGES_REQUESTED** [review]

**Review by jmjuanes — APPROVED** [review]

**Review by imedina — APPROVED** [review]

**Pull request merged**